quickstarts/quickstart-template.md

```diff
@@ -0,0 +1,88 @@
+ 
+
+# Title
+
+:information_source: Before using this template, read the accompanying [quickstart template guide](about-quickstarts.md).
+
+**Document Version:** {MAJOR.MINOR.PATCH} 
+
+**Last Updated:** {"Month DD, YYYY" or "Month YYYY"}   
+
+**Application Version:** 
+
+- {Application 1}: {MAJOR.MINOR.PATCH}
+- {Application 2}: {MAJOR.MINOR.PATCH} or later
+- ...
+
+## Overview
+
+This quickstart guides you through:
+- [Part 1](#part-1-taskname)
+- [Part 2](#part-2-taskname)
+- [Part n](#part-n-taskname)
+
+:information_source: (Optional) Link each part to its corresponding section for easy access
+
+It is intended for {audience}. It assumes that you have basic knowledge of: 
+- Concept 1
+- Concept 2
+- Concept 3 
+
+:information_source: Link each concept to its corresponding content that help users understand the concepts and ideas behind your software. 
+
+## Before you begin
+
+:information_source: This section is optional.
+
+Complete all prerequisites before proceeding with the rest of the quickstart
+
+- Prerequisite 1
+- Prerequisite 2
+- Prerequisite 3
+ 
+## Install
+
+:information_source: This section is optional. Use this section to provide install instructions.
+
+## Part 1: {Task name}
+
+:information_source: Use this section to summarize what users complete in the following steps.
+
+### Step 1: {Step name}
+
+:information_source: Include any code samples, snippets, or screenshots that helps your users to complete the task. If users should see a result after completing this step, make sure to include it.
+
+#### (Optional) Sub step 1: {Substep name}
+
+:information_source: Beware of asking your users to do too much in one step. If needed, break a step into one or two smaller substeps.
+
+## Part 2: {Task name}
+
+...
+
+### Step 2: {Step name}
+
+...
+
+#### (Optional) Sub step 2: {Subtask name}
+
+...
+
+## Part n: {Task name}
+
+...
+
+### Step n: {Step name}
+
+...
+
+#### (Optional) Sub step n: {Subtask name}
+
+...
+
+## Next steps
+
+Now that you've completed this quickstart, try these to learn more about {feature}. 
+- Reference Link 1
+- Reference Link 2
+- Reference Link n
```

Comment:
**Praise:** This template is so awesome, @gayathri-krishnaswamy , Chris, and Nelson! It's very nicely organized, well-expressed and represents some very high quality work. Great work and thank you so much for your contribution!

Reply:
Thanks a ton @barbaricyawps  for your detailed review.  We will work on it and send the next draft soon.